commit 223a238a66cadc1772666582649ed196a302be4e (refs/changes/37/1837/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-25T12:18:54-07:00 (2 years, 5 months ago)
    
    TOOLS-1758 MG: add TRITON jira project to list of eng projects for jira version update scripts

diff --git a/tools/jira/listengprojects.sh b/tools/jira/listengprojects.sh
index 2ab1e3d..283cb86 100755
--- a/tools/jira/listengprojects.sh
+++ b/tools/jira/listengprojects.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2016, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 #
@@ -73,6 +73,7 @@ SKATE
 SSO
 STG
 TOOLS
+TRITON
 VCAPI
 VOLAPI
 WORKFLOW
